doc/build.md

```diff
@@ -29,7 +29,7 @@ Istio Manager requires Go1.8+ toolchain.
 
 Bazel build environment is compatible with the standard Golang tooling, except you need to vendorize all dependencies in Istio Manager. If you have successfully built with Bazel, run the following script to put dependencies fetched by Bazel into `vendor` directory:
 
-    bin/init.sh
+    toolbox/scripts/init.sh
```

Comment:
https://github.com/istio/istio-testing/issues/91
We believe some scripts are doing quite similar jobs and can be shared through different repos. This is just a small test and later we will keep shared stuff only in toolbox and only leave what are special in local repos.